CHANGELOG.rst

```diff
@@ -6,8 +6,9 @@ This document describes changes between each past release.
 1.1 (unreleased)
 ----------------
 
-- Nothing changed yet.
+### Fixed
 
+- Add *.wsgi in the manifest file (#237)
```

Comment:
I would rephrase ans say what problem this fixes. e.g. «Include the wsgi files in the source distribution».